src/main/java/core/mate/academy/service/MachineProducer.java

```diff
@@ -1,11 +1,12 @@
 package core.mate.academy.service;
 
+import core.mate.academy.model.Machine;
 import java.util.List;
 
-public interface MachineProducer {
+public interface MachineProducer<T extends Machine> {
     /**
      * In your implementations - create 2-3 models, add them to the list and return
      * @return - the list of models
      */
-    List<Object> get();
+    List<? extends Machine> get();
```

Comment:
Consider changing the return type of the `get` method to `List<T>` instead of `List<? extends Machine>`. This change will allow implementing classes to return a list of the specific type `T`, which is more flexible and aligns with the generic type parameter defined in the interface.